meta/boardreports/2022-11.md

```diff
@@ -0,0 +1,26 @@
+# Patterns Board Report (Nov)
+
+## Engagement
+
+The [patterns book] is the way inner source practices are captured and shared. Recent web analytics:
+
+* TODO: general trend in traffic (up/down?)
+* TODO: top 3 most popular patterns are? Did pattern of the month have any noticeable effect here?  
+* TODO: any insights from jp language traffic?
+
+## Changes
+
+Changes are contributed via the [InnerSourcePatterns] repository:
+
+* Sebastian Spier hs been very active with various fixes and updates as usual!
+* First-time contribution guide updated for hacktoberfest.
+* New pattern "[Extensions to Manage Contributions at Scale](https://github.com/InnerSourceCommons/InnerSourcePatterns/pull/444/)"
+* Various minor updates to existing patterns.
+
+## Next Goals
+
+* Clear down pending PRs?
+* Expand the content in the book to include more existing but less mature patterns?
```

Comment:
> I would prefer to keep the "included in the booK" as a gatekeeper for a minimum level of quality/maturity.

Very much agree. 

> level up more patterns from Initial to Mature (to get them published in the book).

Yes this is probably a better goal.

These goals as drafted where a rather vague stub tbh, looking back at the goals from last report:

> help the Japanese InnerSource Community to channel new patterns into our repo (in English + Japanese)
> reduce the time that PRs stay in review
> reduce the time it takes from posting a pattern idea to opening a first PR (lower the barrier for entry)
> actively reaching out to orgs that already talk about their InnerSource work publicly. Help them to contribute their patterns. => see [Patterns in the Wild](https://github.com/InnerSourceCommons/InnerSourcePatterns/blob/main/meta/boardreports/2022-05.md#pattern-work)
> hardening the patterns further, by finding 3 known instances for some of the patterns in the book (improves quality and increases trustworthiness of the patterns) => see [Known Instances Collection](https://github.com/InnerSourceCommons/InnerSourcePatterns/blob/main/meta/boardreports/2022-05.md#pattern-work)
> level-up more patterns from the GitHub repo to the book (i.e. from Level 1 to Level 2). The patterns in the book seem to get more exposure.
> trying out other ways to make the patterns easier to discover (e.g. different pattern categories; index pages at the end of the book or similar)
> try out other forms of engagement around patterns (office hours, "pattern of the month", etc)

That's quite a lot of goals 😄 . Broadly I read that as a funnel-ish...

1. increase number of people viewing patterns ("easier to discover", pattern of month, categories)
2. increase number of people thinking to contribute ("patterns in wild", "known instances")
3. increase probability of successful contribution ("reduce review time", "lower barrier of entry")
4. optimise the existing content (level-up more patterns, known instances again)

If you were to pick one...? I tend to be an optimiser sort of person rather than a sociable sort of person so its not that surprising (4) would be my instinct ha.

Reply:
> maybe even list the patterns here where we see the best opportunities to do so

I think these are particularly promising:

* [change-the-middle-management-mindset.md](https://github.com/InnerSourceCommons/InnerSourcePatterns/blob/main/patterns/1-initial/change-the-middle-management-mindset.md)
* [contained-innersource.md](https://github.com/InnerSourceCommons/InnerSourcePatterns/blob/main/patterns/1-initial/contained-innersource.md)
* [developer-incentive-alignment-for-innersource-contribution.md](https://github.com/InnerSourceCommons/InnerSourcePatterns/blob/main/patterns/1-initial/developer-incentive-alignment-for-innersource-contribution.md)
* [extensions-to-manage-contributions-at-scale.md](https://github.com/InnerSourceCommons/InnerSourcePatterns/blob/main/patterns/1-initial/extensions-to-manage-contributions-at-scale.md)
* [governance-levels.md](https://github.com/InnerSourceCommons/InnerSourcePatterns/blob/main/patterns/1-initial/governance-levels.md) ... crap I never finished off [my PR](https://github.com/InnerSourceCommons/InnerSourcePatterns/pull/357). Right, next job. 
* [include-product-owners.md](https://github.com/InnerSourceCommons/InnerSourcePatterns/blob/main/patterns/1-initial/include-product-owners.md)
* [introducing-metrics-in-innersource.md](https://github.com/InnerSourceCommons/InnerSourcePatterns/blob/main/patterns/1-initial/introducing-metrics-in-innersource.md)
* [release-process.md](https://github.com/InnerSourceCommons/InnerSourcePatterns/blob/main/patterns/1-initial/release-process.md)

Reply:
Great list of patterns that are "close to make it to level 2".
The low-hanging fruits here should be the ones that have a Known Instance already, as that info is often hard to get.

That might reduce the list to:

* [extensions-to-manage-contributions-at-scale.md](https://github.com/InnerSourceCommons/InnerSourcePatterns/blob/main/patterns/1-initial/extensions-to-manage-contributions-at-scale.md) - should be quite easy. I recall that we mostly kept this one in Initial to give the community time to review/harden this pattern before it goes into the book
* [governance-levels.md](https://github.com/InnerSourceCommons/InnerSourcePatterns/blob/main/patterns/1-initial/governance-levels.md) - should also be easy, given that we have somebody here who can drive this ;)
* [release-process.md](https://github.com/InnerSourceCommons/InnerSourcePatterns/blob/main/patterns/1-initial/release-process.md) - I don't quite remember why we left this in Initial when we merge it's PR back then. would have to check.
* [contained-innersource.md](https://github.com/InnerSourceCommons/InnerSourcePatterns/blob/main/patterns/1-initial/contained-innersource.md) - this has been around for a long time. best chance might be to activate the two authors again and see if they want to bring it over the finish line.
* **Disclaimer**: I did not check if there are any other Initial patterns that have a Known Instance

Reply:
> If you were to pick one...? I tend to be an optimiser sort of person rather than a sociable sort of person so its not that surprising optimise content would be my instinct ha.

My own instinct is also to optimize. However I feel like I have invested too much time into that area already, and too little in getting more people involved. Therefore I am spending more time on the "sociable" bits these days. This is work in progress though, as I don't feel I/we have found a good way yet to engage more people in this project yet.